mon-pix/app/routes/campaigns/start-or-resume.js

```diff
@@ -86,9 +86,15 @@ export default class StartOrResumeRoute extends Route.extend(SecuredRouteMixin)
   }
 
   get _shouldJoinSimplifiedCampaignAsAnonymous() {
-    return this.state.hasUserSeenLandingPage
-      && this.state.isCampaignSimplifiedAccess
-      && !this.state.isUserLogged;
+    return this.state.isCampaignSimplifiedAccess
+      && !this.state.isUserLogged
```

Comment:
Question : Quelle est la différence avec l'emploi de `this.session.isAuthenticated`, pour savoir si l'utilisateur est connecté ?

Reply:
Aucune (`this.state.isUserLogged = this.session.isAuthenticated`). Dans la route start-or-resume.js, toutes les variables nécessaires à son fonctionnement sont stockés sur l'objet state. La valeur était déjà présente donc on l'a réutilisé.